gems/activerecord/6.0.3.2/activerecord-generated.rbs

```diff
@@ -18051,6 +18051,8 @@ module ActiveRecord
       def relation_class_for: (untyped klass) -> untyped
     end
 
+    def each: () { (untyped) -> void } -> untyped
```

Comment:
`#each` was missing because it is delegated.

https://github.com/rails/rails/blob/6-0-stable/activerecord/lib/active_record/relation/delegation.rb#L88